plugins/out_s3/s3.c

```diff
@@ -994,11 +1034,13 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
 
     if (ctx->compression == FLB_AWS_COMPRESS_GZIP) {
         /* Map payload */
-        ret = flb_aws_compression_compress(ctx->compression, body, body_size, &payload_buf, &payload_size);
+        ret = flb_aws_compression_compress(ctx->compression, body, body_size,
+                                           &payload_buf, &payload_size);
```

Comment:
please check the formatting and spacing here in your editor... it looks a bit weird how github is rendering it

Reply:
the args don't all need to be on separate lines

Reply:
<img width="568" alt="image" src="https://user-images.githubusercontent.com/111381953/211436591-06ddf8f4-4db6-4146-9a58-379844b4bd78.png">
looks fine in VS code